src/test/java/com/server/nodak/domain/vote/service/VoteServiceTest.java

```diff
@@ -0,0 +1,111 @@
+package com.server.nodak.domain.vote.service;
+
+import static com.server.nodak.domain.vote.utils.Utils.createCategory;
+import static com.server.nodak.domain.vote.utils.Utils.createPost;
+import static com.server.nodak.domain.vote.utils.Utils.createUser;
+import static com.server.nodak.domain.vote.utils.Utils.createVote;
+import static com.server.nodak.domain.vote.utils.Utils.createVoteHistory;
+import static com.server.nodak.domain.vote.utils.Utils.createVoteOption;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+
+import com.server.nodak.domain.post.domain.Category;
+import com.server.nodak.domain.post.domain.Post;
+import com.server.nodak.domain.user.domain.User;
+import com.server.nodak.domain.user.repository.UserRepository;
+import com.server.nodak.domain.vote.domain.Vote;
+import com.server.nodak.domain.vote.domain.VoteHistory;
+import com.server.nodak.domain.vote.domain.VoteOption;
+import com.server.nodak.domain.vote.dto.VoteOptionResult;
+import com.server.nodak.domain.vote.dto.VoteResultResponse;
+import com.server.nodak.domain.vote.repository.VoteHistoryRepository;
+import com.server.nodak.domain.vote.repository.VoteOptionRepository;
+import com.server.nodak.domain.vote.repository.VoteRepository;
+import java.util.List;
+import java.util.Optional;
+import java.util.Random;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@DisplayName("VoteService 테스트")
+@Slf4j
+class VoteServiceTest {
+    Random rnd = new Random();
+    @InjectMocks
+    VoteService voteService;
+    @Mock
+    UserRepository userRepository;
+    @Mock
+    VoteRepository voteRepository;
+    @Mock
+    VoteOptionRepository voteOptionRepository;
+    @Mock
+    VoteHistoryRepository voteHistoryRepository;
+    User user;
```

Comment:
뭐가 정답인지는 모르겠지만, 위처럼 코드를 너무 붙여 써도 답답해 보이긴 하네요.
저희가 플젝을 실제처럼 하고 싶다면, 코드 컨벤션도 어느 정도 지키면 좋을 것 같습니다!

```suggestion
    Random rnd = new Random();
    
    @InjectMocks
    VoteService voteService;
    @Mock
    UserRepository userRepository;
    @Mock
    VoteRepository voteRepository;
    @Mock
    VoteOptionRepository voteOptionRepository;
    @Mock
    VoteHistoryRepository voteHistoryRepository;
    
    User user;
```

Reply:
답답해 보이긴 하네요 ! 수정하겠습니다 :)